client/api/filesystem/v1beta2/api.proto

```diff
@@ -5,22 +5,22 @@ package v1beta2;
 option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1beta2";
 
 service Filesystem {
-    // PathExists checks if the requested path exists in the host's filesystem
+    // PathExists checks if the requested path exists in the host filesystem.
     rpc PathExists(PathExistsRequest) returns (PathExistsResponse) {}
 
-    // Mkdir creates a directory at the requested path in the host's filesystem
+    // Mkdir creates a directory at the requested path in the host filesystem.
     rpc Mkdir(MkdirRequest) returns (MkdirResponse) {}
 
-    // Rmdir removes the directory at the requested path in the host's filesystem.
-    // This may be used for unlinking a symlink created through LinkPath
+    // Rmdir removes the directory at the requested path in the host filesystem.
+    // This may be used for unlinking a symlink created through CreateSymlink.
     rpc Rmdir(RmdirRequest) returns (RmdirResponse) {}
 
-    // LinkPath creates a local directory symbolic link between a source path
-    // and target path in the host's filesystem
-    rpc LinkPath(LinkPathRequest) returns (LinkPathResponse) {}
+    // CreateSymlink creates a local directory symbolic link from a target path
```

Comment:
creates a symbolic link, target_path pointing to source_path in the host filesystem.
Also I am thinking to name target_path to link_path to avoid confusion. What do you think?

Reply:
target is really confusing in many places. https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createsymboliclinka

Reply:
I think that target_path and source_path are ok, from the documentation it's pretty clear that target_path is pointing to source_path, I interpret the arrow like this (from target path to source path) because that's what `ls` says:

```
/tmp ❯ touch foo
/tmp ❯ ln -s foo bar
/tmp ❯ ls -la foo bar
lrwxr-xr-x  1 mauriciopoppe  wheel  3 Jun  4 11:53 bar -> foo
-rw-r--r--  1 mauriciopoppe  wheel  0 Jun  4 11:53 foo
```

Reply:
ok, then update comment a little bit with
```
create a symbolic link called target_path that points to source_path in the host filesystem (target_path is the name of the symbolic link created, source_path is the existing path).
```

Reply:
done